web/themes/contrib/civictheme/includes/form_element.inc

```diff
@@ -352,20 +361,29 @@ function _civictheme_preprocess_form_element__generic(array &$variables): void {
   $variables['title'] = $element['#title'] ?? '';
 
   $title_display = $element['#_title_display'] ?? $element['#title_display'] ?? $variables['title_display'] ?? 'visible';
-  $title_display = $title_display == 'none' ? 'hidden' : $title_display;
-  $title_display = $title_display == 'visually_hidden' ? 'invisible' : $title_display;
+  $title_display = $title_display === 'none' ? 'hidden' : $title_display;
+  $title_display = $title_display === 'visually_hidden' ? 'invisible' : $title_display;
   if (empty($variables['title'])) {
     $title_display = 'hidden';
   }
   $variables['title_display'] = $title_display;
 
-  $variables['orientation'] = $variables['orientation'] ?? $title_display == 'inline' ? 'horizontal' : 'vertical';
+  $variables['orientation'] = $variables['orientation'] ?? $title_display === 'inline' ? 'horizontal' : 'vertical';
 
   $variables['is_required'] = $element['#required'] ?? FALSE;
   $variables['is_disabled'] = isset($element['#attributes']['disabled']);
   $variables['description'] = $element['#description'] ?? '';
-  $variables['message'] = $element['#errors'] ?? '';
-  $variables['is_invalid'] = isset($element['#errors']) && !empty($element['#errors']);
+  $variables['is_invalid'] = !empty($element['#errors']);
+  if ($variables['is_invalid']) {
+    $variables['attributes']['aria-invalid'] = 'true';
+    $variables['message']['content'] = $element['#errors'] ?? '';
+    $element_id = $element['#id'] ?? $element['#attributes']['data-drupal-selector'] ?? '';
+    if (!empty($element_id)) {
+      $element_id .= '--error-message';
+      $variables['message']['attributes'] = new Attribute(['id' => $element_id]);
```

Comment:
Note we add the ID for the message and the aria-described-by for the form element